galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -453,3 +453,12 @@ messages:
     level: Warning
     description: "Application pods should not run as user ID (UID) 1337"
     template: "User ID (UID) 1337 is reserved for the sidecar proxy."
+
+  - name: "ConflictingGateways"
+    code: IST0145
+    level: Error
+    description: "Gateway should not have the same selector, port number and matched hosts of server"
+    template: "Conflicting gateways are detected for gateways %s on selector, port number and hosts."
```

Comment:
```suggestion
    template: "Conflicting gateways are detected for gateway %s on selector, port number and hosts."
```

Reply:
okay, I will change it to make them to be unified